### ISSUE TITLE
System Tray missing Ubuntu 16.04 LTS

### ISSUE THREAD

**Issue by tank-trax** [symless/synergy-core#6544]
Am currently using 
  
Server: Windows 10 
Client: Ubuntu 16.04.2
Desktop is XFCE
Synergy version: 1.10.2

The problem is with the most recent Linux version for Ubuntu 16.04 LTS

I installed synergy_1.10.2.stable_b100+8c010140_ubuntu16_amd64.deb and it runs well but there is no system tray icon and no way that I can see in the settings to force it to show up there

fortunately I kept older versions for Windows and Linux and reverted back to synergy-v1.8.8-stable-25a8cb2-Linux-x86_64.deb which has the Synergy Icon in the system tray

to recap... the older version has the tray icon, the newer version doesn't
